### ISSUE TITLE
Multiselect for files/folders within drives

### ISSUE THREAD

**Issue by RaXorX** [kapitainsky/RcloneBrowser#121]
Another enhancement features, not sure if it is fixed in this current beta or not. Multi select wouldn't just be beneficial in tasks menu only. I noticed when I am inside the drives I am unable to select multiple files at the moment in 1.8.0 version. This limits me to either have to make a folder first, move the files I want (from the actual browser since I again can't select multiple files) and then download that folder from rclone browser. A tedious task which could be shortened.

**Comment by dia3olik** [comment]
yeah I agree, there is another identical thread I opened, try writing on that one too

**Comment by kapitainsky** [comment]
It is definitely high on my priority list. Recently could not spend much time on RB but now will start working on it again
